test/src/org/labkey/test/tests/panoramapremium/TargetedMSHidePeptidesAndMolecules.java

```diff
@@ -78,6 +79,30 @@ public void testPeptideAndMoleculeExclusionAndInclusion()
         excludePeptideOrMolecule(MoleculesOnlySubFolder, 0, "Included");
     }
 
+    /*
+        Test coverage for : Issue 46048: Configure Included and Excluded Precursors - blank or duplicate precursors in some scenarios
```

Comment:
This is good and covers half of the behavior. It would be ideal to verify that you get a helpful error message when there is no data imported into the current folder. Maybe the test could create its own subfolder without importing any data to hit this case too.

Reply:
Done